### DISCUSSION TITLE
[firebase_database] Not plan for web ?

### DISCUSSION THREAD

**Discussion by tperraut** [firebase/flutterfire#4276] [Feature request] [answered]
As I saw on the roadmap here #2582 it seems there is not plan for firebase_datase support on web. Why is so ?

Any chance to have it planned or where should I start to help adding support for it ?

Thanks for your work.

**Comment by deepak786** [comment, marked as answer]
@tperraut PR https://github.com/FirebaseExtended/flutterfire/pull/6952 is already in Progress for web support.
